index.html

```diff
@@ -118,6 +118,33 @@ <h2>A project for first-time contributions</h2>
       <!-- Keep one line of space above and below your card -->
       <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________  qianqiu card START  ________  -->
+        <div class="card">
+          <p class="name">qianqiu</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://www.twitter.com/your_user_handle" target="_blank">Your handle</a>
```

Comment:
If you have an x account, please update the default link and text with your x link and name.